uiowa/uiowa
# Feature: Update lb_direct_add module and related styles.

This PR tracks changes going on in [this issue](https://www.drupal.org/project/lb_direct_add/issues/3157440). 

Resolves #3244.

# How to test

* `blt ds`
* `drush @default.local uli node/1/layout` and observe popover functionality for "Add blocks".
* Ensure that clicking block links and "More..." link still works.
* Visit http://sandbox.local.drupal.uiowa.edu/admin/config/content/layout-builder-direct-add to view the new settings page.
